bittensor/subtensor.py

```diff
@@ -4384,9 +4499,10 @@ def make_substrate_call_with_retry():
             )
 
         json_body = make_substrate_call_with_retry()
-        result = json_body["result"]
+        result = json_body.get("result", None)
 
-        if result in (None, []):
+        if not result:
```

Comment:
Non-blocking: you could simplify this with walrus expressions, but it's not *needed*.

Reply:
Sure. Done everywhere within the module.
But it is worth remembering that this expression complicates the understanding and readability of the code.